src/Model/SalesOrder/SalesOrderForm.php

```diff
@@ -37,6 +37,11 @@ class SalesOrderForm
 	 * @var string
 	 */
 	private $transactionDate;
+    /**
+     * @SerializedName("custbody_rsm_so_type")
+     * @var string
+     */
+    private $salesOrderType;
```

Comment:
Since the class `SalesOrderForm` represents a Sales Order, isn't "salesOrder" prefix redundant?
Would it be too generic to call this field `$type`?

Reply:
Fair point, fix comming right up